src/main/java/com/velox/sloan/cmo/workflows/IgoLimsPluginUtils/AliquotingTags.java

```diff
@@ -0,0 +1,50 @@
+package com.velox.sloan.cmo.workflows.IgoLimsPluginUtils;
+
+public interface AliquotingTags {
+
+    public static final String ALIQUOT_STARTING_CONCENTRATION = ".*<!--\\s*ALIQUOT\\s*STARTING\\s*CONCENTRATION\\s*-->.*";
```

Comment:
'public static final' redundant for interface, let's remove it.

Reply:
got it.

Reply:
I applied the changes as suggested above.